kinect2_description/urdf/kinect2/kinect2-gazebo.xacro

```diff
@@ -80,8 +80,8 @@
                 <plugin name="${name}_camera_controller"
                         filename="libgazebo_ros_camera.so">
                     <alwaysOn>true</alwaysOn>
-                    <update_rate>30.0</update_rate>
-                    <cameraName>${name}</cameraName>
+                    <updateRate>30.0</updateRate>
+                    <cameraName>${name}_camera</cameraName>
```

Comment:
I agree with @MFernandezCarmona that the names should resemble the real sensor as much as possible.